crates/scroll/evm/src/execute.rs

```diff
@@ -304,34 +313,49 @@ mod tests {
 
         // determine l1 gas oracle storage
         let l1_gas_oracle_storage =
-            if strategy.spec().is_feynman_active_at_timestamp(block_timestamp) {
+            if strategy.spec().is_galileo_active_at_timestamp(block_timestamp) {
```

Comment:
Do we want to add a case for Galileo V2?

Reply:
I think this whole test needs a major refactor. But for now, I added only `galileo` because that is where the fee formula changes, it does not change in `galileoV2`.